README.md

```diff
@@ -7,20 +7,20 @@
 <br>
 
 <p align="center">
-   <a href="https://github.com/0xPolygonID/docs/network/members"><img src="https://img.shields.io/github/forks/0xPolygonID/docs?style=social"></a>
-   <img src="https://img.shields.io/github/stars/0xPolygonID/docs?style=social">
-   <a href="https://x.com/PrivadoID"><img src="https://img.shields.io/twitter/follow/0xPolygonID.svg?style=social"></a>
+   <a href="https://github.com/optimismID/docs/network/members"><img src="https://img.shields.io/github/forks/optimismID/docs?style=social"></a>
+   <img src="https://img.shields.io/github/stars/optimismID/docs?style=social">
+   <a href="https://x.com/PrivadoID"><img src="https://img.shields.io/twitter/follow/optimismID.svg?style=social"></a>
    <br>
-   <img src="https://img.shields.io/github/languages/count/0xPolygonID/docs">
-   <a href="https://github.com/0xPolygonID/docs/issues"><img src="https://img.shields.io/github/issues/0xPolygonID/docs"></a>
-   <a href="https://github.com/0xPolygonID/docs/pulls"><img src="https://img.shields.io/github/issues-pr-raw/0xPolygonID/docs"></a>
-   <a href="https://github.com/0xPolygonID/docs/graphs/contributors"><img src="https://img.shields.io/github/contributors-anon/0xPolygonID/docs"></a>
-   <img src="https://img.shields.io/github/languages/code-size/0xPolygonID/docs">
+   <img src="https://img.shields.io/github/languages/count/optimismID/docs">
+   <a href="https://github.com/optimismID/docs/issues"><img src="https://img.shields.io/github/issues/optimismID/docs"></a>
+   <a href="https://github.com/optimismID/docs/pulls"><img src="https://img.shields.io/github/issues-pr-raw/optimismID/docs"></a>
+   <a href="https://github.com/optimismID/docs/graphs/contributors"><img src="https://img.shields.io/github/contributors-anon/optimismID/docs"></a>
```

Comment:
All this are links, we should point to the right file, a plain replace just breaks the links